packages/ui/src/components/Radio/index.tsx

```diff
@@ -138,7 +140,17 @@ type RadioProps = {
     | 'id'
     | 'name'
     | 'required'
-  >
+  > &
+  (
+    | {
+        'aria-label': string
```

Comment:
This is not typed when we do that right ?

Reply:
should we use ariaLabel as props when we control usage with children ?

Reply:
It's typed, it's an XOR (the XOR custom type break eslint idk why). The aria-label prop can't be used when `children` is provided.